index.md

```diff
@@ -3,21 +3,25 @@ layout: default
 title: SapMachine
 ---
 
-**SapMachine** is a free, open-source, non-commercial, cross-platform, and fully-functional production-grade version of the Open Java Development Kit (OpenJDK).
-Built and supported by SAP, it provides a robust and reliable Java environment.
+**SapMachine** is the free, multiplatform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility) [OpenJDK](https://openjdk.org/) (Open Java Development Kit) distribution by [SAP](https://sap.com). It serves as the default JDK for SAP's countless applications and services.
 
 <img align="left" width="240" src="assets/images/logo_circular.svg" alt="Logo of SapMachine">
 
-Following the release cadence of the OpenJDK, a new feature release is shipped every six months, with one designated as a long-term support release every two years.
-Update releases for active versions are provided every quarter. So SapMachine offers long-term support, regular performance improvements and timely security updates.
+This Distribution supports all major Operation Systems.
```

Comment:
It's still "Operation Systems" instead of "Operating Systems"

Reply:
Good catch :)